src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -425,15 +425,13 @@ private void BuildProject()
 
                 BuildProjectWithMsbuild();
             }
-            else
-            {
-                TargetPath = ResolveTargetPath();
 
-                // Make if the target path doesn't exist, fail
-                if (!Directory.Exists(TargetPath) && !File.Exists(TargetPath))
-                {
-                    throw new PackagingException(NuGetLogCode.NU5012, String.Format(CultureInfo.CurrentCulture, LocalizedResourceManager.GetString("UnableToFindBuildOutput"), TargetPath));
-                }
+            TargetPath = ResolveTargetPath();
+
+            // Make if the target path doesn't exist, fail
+            if (!Directory.Exists(TargetPath) && !File.Exists(TargetPath))
```

Comment:
```suggestion
            if (!File.Exists(TargetPath))
```

Reply:
Here `TargetPath` could be either be a file or a directory path. So both checks are needed to validate if we get either one.